{"links": {"self": {"href": "data/repositories/osrf/subt/issues/120/comments/53115553.json"}, "html": {"href": "#!/osrf/subt/issues/120#comment-53115553"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/subt/issues/120.json"}}, "type": "issue", "id": 120, "repository": {"links": {"self": {"href": "data/repositories/osrf/subt.json"}, "html": {"href": "#!/osrf/subt"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3c95f6ad-c304-407c-b838-09597d836552}ts=2272898"}}, "type": "repository", "name": "subt", "full_name": "osrf/subt", "uuid": "{3c95f6ad-c304-407c-b838-09597d836552}"}, "title": "Transform from X2_1/base_link/imu_sensor to X2_1/base_link is unavailable"}, "content": {"raw": "Thank you very much for the advice. Unfortunately it does not solve my problem. I added the parameter in my launch-file but still the messages occur. \n\nIn a second attempt I checked the parameter directly after starting the tunnel world \\(with spawning my robot\\):\n\n`:~/workspace/subt_seed_ws$ rosparam get /use_sim_time`  \n`true`\n\n After some attempts I got many messages like these:\n\n`ESC[33m[ WARN] [1563915818.701178137, 6.801000000]: Detected jump back in time of 0.003s. Clearing TF buffer.ESC[0m`  \n`ESC[33m[ WARN] [1563915818.706968957, 6.816000000]: Detected jump back in time of 0.001s. Clearing TF buffer.ESC[0m`  \n`ESC[33m[ WARN] [1563915818.709302017, 6.802000000]: Detected jump back in time of 0.014s. Clearing TF buffer.ESC[0m`  \n`ESC[33m[ WARN] [1563915818.718298978, 6.816000000]: Detected jump back in time of 0.018s. Clearing TF buffer.ESC[0m`  \n`ESC[33m[ WARN] [1563915818.732165092, 6.802000000]: Detected jump back in time of 0.035s. Clearing TF buffer.ESC[0m`  \n`ESC[33m[ WARN] [1563915818.769012807, 6.842000000]: Could not obtain transform from base_link to X2_1/base_link. Error was \"base_link\" passed to lookupTransform argument source_frame does not exist.`  \n`ESC[0m`  \n`ESC[33m[ WARN] [1563915818.770330282, 6.840000000]: Detected jump back in time of 0.002s. Clearing TF buffer.ESC[0m`  \n`ESC[33m[ WARN] [1563915818.785496980, 6.840000000]: Detected jump back in time of 0.004s. Clearing TF buffer.ESC[0m`  \n`ESC[33m[ WARN] [1563915818.859838999, 6.840000000]: Could not obtain transform from base_link to X2_1/base_link. Error was \"base_link\" passed to lookupTransform argument source_frame does not exist.`  \n`ESC[0m`  \n`ESC[33m[ WARN] [1563915818.861674419, 6.840000000]: Detected jump back in time of 0.006s. Clearing TF buffer.ESC[0m`  \n`ESC[33m[ WARN] [1563915818.879973618, 6.850000000]: Detected jump back in time of 0.001s. Clearing TF buffer.ESC[0m`  \n`ESC[33m[ WARN] [1563915818.891442905, 6.853000000]: Transform from X2_1/base_link/imu_sensor to X2_1/base_link was unavailable for the time requested. Using latest instead.`  \n`ESC[0m`  \n`ESC[33m[ WARN] [1563915818.901706298, 6.853000000]: Detected jump back in time of 0.001s. Clearing TF buffer.ESC[0m`\n\nSomething must have changed in Ignition because the same code still works in Gazebo 9.\n\nDo you have another idea?", "markup": "markdown", "html": "<p>Thank you very much for the advice. Unfortunately it does not solve my problem. I added the parameter in my launch-file but still the messages occur. </p>\n<p>In a second attempt I checked the parameter directly after starting the tunnel world (with spawning my robot):</p>\n<p><code>:~/workspace/subt_seed_ws$ rosparam get /use_sim_time</code><br />\n<code>true</code></p>\n<p>After some attempts I got many messages like these:</p>\n<p><code>ESC[33m[ WARN] [1563915818.701178137, 6.801000000]: Detected jump back in time of 0.003s. Clearing TF buffer.ESC[0m</code><br />\n<code>ESC[33m[ WARN] [1563915818.706968957, 6.816000000]: Detected jump back in time of 0.001s. Clearing TF buffer.ESC[0m</code><br />\n<code>ESC[33m[ WARN] [1563915818.709302017, 6.802000000]: Detected jump back in time of 0.014s. Clearing TF buffer.ESC[0m</code><br />\n<code>ESC[33m[ WARN] [1563915818.718298978, 6.816000000]: Detected jump back in time of 0.018s. Clearing TF buffer.ESC[0m</code><br />\n<code>ESC[33m[ WARN] [1563915818.732165092, 6.802000000]: Detected jump back in time of 0.035s. Clearing TF buffer.ESC[0m</code><br />\n<code>ESC[33m[ WARN] [1563915818.769012807, 6.842000000]: Could not obtain transform from base_link to X2_1/base_link. Error was \"base_link\" passed to lookupTransform argument source_frame does not exist.</code><br />\n<code>ESC[0m</code><br />\n<code>ESC[33m[ WARN] [1563915818.770330282, 6.840000000]: Detected jump back in time of 0.002s. Clearing TF buffer.ESC[0m</code><br />\n<code>ESC[33m[ WARN] [1563915818.785496980, 6.840000000]: Detected jump back in time of 0.004s. Clearing TF buffer.ESC[0m</code><br />\n<code>ESC[33m[ WARN] [1563915818.859838999, 6.840000000]: Could not obtain transform from base_link to X2_1/base_link. Error was \"base_link\" passed to lookupTransform argument source_frame does not exist.</code><br />\n<code>ESC[0m</code><br />\n<code>ESC[33m[ WARN] [1563915818.861674419, 6.840000000]: Detected jump back in time of 0.006s. Clearing TF buffer.ESC[0m</code><br />\n<code>ESC[33m[ WARN] [1563915818.879973618, 6.850000000]: Detected jump back in time of 0.001s. Clearing TF buffer.ESC[0m</code><br />\n<code>ESC[33m[ WARN] [1563915818.891442905, 6.853000000]: Transform from X2_1/base_link/imu_sensor to X2_1/base_link was unavailable for the time requested. Using latest instead.</code><br />\n<code>ESC[0m</code><br />\n<code>ESC[33m[ WARN] [1563915818.901706298, 6.853000000]: Detected jump back in time of 0.001s. Clearing TF buffer.ESC[0m</code></p>\n<p>Something must have changed in Ignition because the same code still works in Gazebo 9.</p>\n<p>Do you have another idea?</p>", "type": "rendered"}, "created_on": "2019-07-23T21:15:09.405878+00:00", "user": {"display_name": "Sophisticated Engineering", "uuid": "{12ed2357-e253-4c67-a2b9-7e12c64773e4}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B12ed2357-e253-4c67-a2b9-7e12c64773e4%7D"}, "html": {"href": "https://bitbucket.org/%7B12ed2357-e253-4c67-a2b9-7e12c64773e4%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/2a19963da6b0f34b3be28d8690bdd711d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsSE-6.png"}}, "nickname": "sopheng", "type": "user", "account_id": "5c0d82b3128c7106f57691a7"}, "updated_on": null, "type": "issue_comment", "id": 53115553}